### PR TITLE
Do not reset mineTile of units to null if client

### PR DESCRIPTION
**Pull request by LeoSko**
That allows server to put any tile into `mineTile` while client still draws it perfectly fine. Without that, client resets it regardless of server's data.
This way some ~~crazy~~ stuff can be allowed by servers if they want to, like
Анимация

### CODE REVIEW

**Comment by oinkymoo** [comment]
can you explain what this means in layman's terms?

**Comment by LeoSko** [comment]
@oinkymoo, gif pretty much explains that. Note that it can apply to custom servers, not to vanilla game.

**Comment by oinkymoo** [comment]
basically, lets servers tell units to mine tiles they shouldn't

**Comment by LeoSko** [comment]
Yes.

**Comment by oinkymoo** [comment]
cool thank
